scripts/zones/Davoi/mobs/Davoi_Mush.lua

```diff
@@ -1,31 +1,14 @@
 -----------------------------------
 -- Area: Davoi
 --  MOB: Davoi Mush
+-- Note: PH for Blubbery Bulge
 -----------------------------------
-
-require("scripts/globals/settings");
-
------------------------------------
--- onMobDeath
------------------------------------
+require("scripts/zones/Davoi/MobIDs");
+require("scripts/globals/mobs");
 
 function onMobDeath(mob, player, isKiller)
 end;
 
------------------------------------
--- onMobDespawn
------------------------------------
-
 function onMobDespawn(mob)
-    local mobID = mob:getID();
-    -- Wiki doesn't mention any sort of cooldown, so assuming pure lotto for now.
-    if (mobID == 17387919) then -- Making sure is the correct mush.
-        if (math.random(0,99) > 15) then -- Guesstimated 15% chance of pop.
-            local BlubberyBulge = 17387920;
-            DisallowRespawn(BlubberyBulge, false);
-            GetMobByID(BlubberyBulge):setRespawnTime(GetMobRespawnTime(mobID));
-            SetServerVariable("[PH]Blubbery_Bulge", mobID);
-            DisallowRespawn(mobID, true);
-        end
-    end
-end;
\ No newline at end of file
+    phOnDespawn(mob,BLUBBERY_BULGE_PH,20,3600); -- 1 hour
```

Comment:
you found cooldown info for him?

Reply:
Meh actually I'm fine with the assumed 1hr instead, coz who camps this guy seriously anyway..carry on!